exercises/split-second-stopwatch/instructions.md

```diff
@@ -0,0 +1,22 @@
+# Instructions
```

Comment:
Should the `advance time` be mentioned in the instructions?

Reply:
Not in the instructions I think, as it is not something the students need to be bothered with. But I've added it to the canonical data.

Reply:
I'm fine either way, but I'm confused by "not something the students need to be bothered with". Won't the students need to implement handling this?